packages/@aws-cdk/core/README.md

```diff
@@ -305,6 +306,54 @@ new CustomResource(this, 'MyResource', {
 });
 ```
 
+#### The `core.CustomResourceProvider` class
```

Comment:
Somewhere in this readme, add a section on when to use the slim class vs. the full blown framework, and why there are two different options.

Reply:
Added